src/test/java/org/elasticsearch/rest/action/admin/indices/upgrade/UpgradeTest.java

```diff
@@ -246,6 +243,7 @@ public static void runUpgrade(HttpRequestBuilder httpClient, String index, Strin
         assertEquals(200, rsp.getStatusCode());
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
it's perfectly fine to clean these things up as we go... I don't see a need to have separate PRs for these small cleanups